model/uzivatel.php

```diff
@@ -261,6 +261,49 @@ function jmeno()
     return $this->u['jmeno_uzivatele'].' '.$this->u['prijmeni_uzivatele'];
   }
 
+  /** Jméno uživatele */
+  function getJmeno()
```

Comment:
style: neprefixovat get, `function nazevMetody() {`

Obecná myšlenka: pokud třída uživatel má nějakou rozumnou metodu na zobrazení (jenom) občanského jména, zkusil bych ji použít, nebo vytvořil, pokud nemá, a měnil bych ve view jméno a příjmení najednou. Principy: 

- méně metod (poukud to umí stejnou věc a nezhoršuje čitelnost) lépe (resp. kratší kód)
- snažím se napsat co nejkonkrétnější metodu na to, co ve view potřebuji vědět (informaci), než předávat čistá data, a jejich zpracování (převod data -> informace/znalost) řešit ve view.
- proč obecně mít get metody 1:1 s sloupci databáze je špatný nápad rozsáhleji viz https://www.martinfowler.com/bliki/AnemicDomainModel.html